cpp/src/parquet/properties.h

```diff
@@ -120,6 +123,12 @@ class PARQUET_EXPORT ReaderProperties {
     page_checksum_verification_ = check_crc;
   }
 
+  // Set the default read size to read the footer from a file. For high latency
+  // file systems and files with large metadata (>64KB) this can increase performance
+  // by reducing the number of round-trips to retrieve the entire file metadata.
+  size_t footer_read_size() const { return footer_read_size_; }
+  void set_footer_read_size(size_t size) { footer_read_size_ = size; }
```

Comment:
Not an expert here but will this generate the proper documentation? This will add this documentation to the getter instead of the setter, right?
https://arrow.apache.org/docs/cpp/api/formats.html#_CPPv4N7parquet16ReaderProperties26file_decryption_propertiesENSt10shared_ptrI24FileDecryptionPropertiesEE

Reply:
I agree that users are more interested in setter than getter. Therefore adding a doc string for setter is useful.

Reply:
updated